tests/filters.rs

```diff
@@ -0,0 +1,79 @@
+//! Tests for --exclude --include flags for suckit
```

Comment:
This is a comment for the whole file. The code is good, but maybe a little more spacing and 1 or 2 comments will greatly improve the readability

Reply:
`cargo fmt` is handling the white spacing convention. I have added a few comments over the test methods. Does that improve readability?

Reply:
I was talking about vertical white space, but that's better now, thanks